tools/sdk-readme-table-generator/TableGenerator/Shared.fs

```diff
@@ -6,11 +6,13 @@ open System
 type Branch =
     { GitBranchName: string
       DisplayName: string
-      AkaMsChannel: string option }
+      AkaMsChannel: string option 
+      UseLegacyAkaMS: bool }
```

Comment:
can me make the naming consistent, similar to AkaMsChannel (instead of AkaMS)?

Reply:
Done